frontend/cypress/e2e/patientEntry.cy.js

```diff
@@ -43,7 +43,7 @@ describe('Patient Search', function () {
         cy.wait(200)
         patientPage.clickSearchPatientButton();
         cy.fixture('Patient').then((patient) => {
-            patientPage.validatePatientSearchTable(patient.firstName, patient.inValidName);
+            patientPage.validatePatientByGender("M");
         });
         cy.wait(200).reload();
     });
```

Comment:
For better testing approach, we need to add a configuration that will have to trigger these tests automatically once the PR is pushed.  That might quicken both e2e tests and other work. cc @mozzy11

Reply:
I agree , have we used this [github action](https://github.com/cypress-io/github-action)

@mozzy11